Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "near-lake"
-version = "0.1.19"
+version = "0.1.20-rc.0"
```

Comment:
Is creating a release candidate version the right thing to do in this situation?

Reply:
Yeah, why not?

Reply:
I'm just not sure what the right process is, I'm not against it :)